account_test.go

```diff
@@ -0,0 +1,290 @@
+package dcrlibwallet_test
+
+import (
+	"context"
+	"encoding/json"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+
+	w "github.com/decred/dcrwallet/wallet/v3"
+	"github.com/decred/dcrwallet/wallet/v3/udb"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/raedahgroup/dcrlibwallet"
+)
+
+const rootDir = ".dcrlibwallet_tests_root"
+
+func init() {
+	SetLogLevels("error")
+}
+
+var _ = Describe("Account", func() {
+	var (
+		wallet         *Wallet
+		internalWallet *w.Wallet
+		password       string
+	)
+
+	BeforeSuite(func() {
+		os.RemoveAll(rootDir)
+		multi, err := NewMultiWallet(rootDir, "", "testnet3")
+		Expect(err).To(BeNil())
+		Expect(multi).ToNot(BeNil())
+
+		password = randomPassword()
+		wallet, err = multi.CreateNewWallet(password, 0)
+		Expect(err).To(BeNil())
+		Expect(wallet).ToNot(BeNil())
+		internalWallet = wallet.InternalWallet()
+	})
+
+	AfterSuite(func() {
+		if wallet != nil {
+			wallet.Shutdown()
+		}
+	})
```

Comment:
Since `BeforeSuite` and `AfterSuite` run before and after ALL test blocks, I think it's more readable to place these outside the `Describe` block - they should have no parent block.